src/pages/patientView/trialMatch/style/trialMatch.module.scss

```diff
@@ -64,7 +64,7 @@
   .firstRight{
     display:flex;
     flex: 0 0 auto;
-    width: 300px;
+    width: 50%;
```

Comment:
the PR title says 100%, but we set it here to 50%. Are they two unrelated things?

Reply:
The default width is 50% but it will be set as 100% when there is no genomic content.